perf-tool/src/client.cpp

```diff
@@ -166,16 +191,21 @@ void Client::closeClient()
 int main(int argc, char const *argv[])
 {
     string hostname = "localhost";
+    std::string clientLogPath = "clientLogs.json";
     int portno = ServerConstants::DEFAULT_PORT;
 
     if (argc > 1)
     {
         portno = atoi(argv[1]);
         if (argc > 2)
-            hostname = argv[2];
+        {
+            clientLogPath = argv[2];
+            if (argc > 3)
+                hostname = argv[3];
+        }
```

Comment:
Solution is not ideal because if I want to specify the host where the target JVM is running, I must specify the log file as well. Ideally we would have option switches like: `--hostname localhost --log logname`